src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -579,6 +588,28 @@ public void TestCapitalize()
             Assert.AreEqual("", StandardFilters.Capitalize(""));
             Assert.AreEqual(" ", StandardFilters.Capitalize(" "));
             Assert.AreEqual("That Is One Sentence.", StandardFilters.Capitalize("That is one sentence."));
+        }
+		
+		[Test]
+        public void TestUniq()
+        {
+			CollectionAssert.AreEqual(new[] { "ants", "bugs", "bees" }, StandardFilters.Uniq(new string[] { "ants", "bugs", "bees", "bugs", "ants" }));
+            CollectionAssert.AreEqual(new string[] {}, StandardFilters.Uniq(new string[] {}));
+            Assert.AreEqual(null, StandardFilters.Uniq(null));
+			Assert.AreEqual(new List<object> {5}, StandardFilters.Uniq(5));
+        }
+		
+		[Test]
+        public void TestAbs()
+        {
+			Assert.AreEqual(0, StandardFilters.Abs("notNumber"));
+            Assert.AreEqual(10, StandardFilters.Abs(10));
+            Assert.AreEqual(5, StandardFilters.Abs(-5));
+            Assert.AreEqual(19, StandardFilters.Abs(19.86));

```

Comment:
`Abs` shouldn't round a number as I understand from the [liquid doc](https://shopify.github.io/liquid/filters/abs/).

Reply:
yes, my bad. Fixing soon.